src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -356,6 +358,7 @@ function TTYGViewCtrl(
             templateUrl: 'js/angular/ttyg/templates/modal/agent-settings-modal.html',
             controller: 'AgentSettingsModalController',
             windowClass: 'agent-settings-modal',
+            backdrop: 'static',
```

Comment:
Wouldn't it be better if this implementation detail is hidden in the Modal service?

eg. have another prop `closeOnBackdrop` which would internally set `backdrop: 'static'`